user_acceptance_tests/deploy_configs/happy-aws-debian.deploy.config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "services": [
+      {
+        "id": "java1",
+        "source_repository": "-b debian-support https://github.com/newrelic/demo-javatron.git",
```

Comment:
Will need to remove the branch ('debian-support') from this line before this PR gets merged.

Reply:
this can just be `"local_source_path": "demo-javatron"`, which is how the other configs are set up, unless im mistaken.